cdp_scrapers/legistar_utils.py

```diff
@@ -1046,12 +1046,20 @@ def date_time_to_datetime(ev_date: str, ev_time: str) -> datetime:
         # 2021-07-09T00:00:00
         d = datetime.strptime(ev_date, "%Y-%m-%dT%H:%M:%S")
         # 9:30 AM
-        t = datetime.strptime(ev_time, "%I:%M %p")
-        return datetime(
-            year=d.year,
-            month=d.month,
-            day=d.day,
-            hour=t.hour,
-            minute=t.minute,
-            second=t.second,
-        )
+        # for kingCounty, some events have ev_time =None
```

Comment:
I would generalize this comment. This is the legistar utils so dont mention king county just mention that its possible that ev_time can be none.